### PR TITLE
Fix for Backed By, Achievements and Trusted By Logo should be english version

### PR DESCRIPTION
**Pull request by mailtime-wing**
Added english version logos in backed by and trusted by section

### CODE REVIEW

File: src/components/LogosSection/index.js [hunk 1]
```diff
@@ -128,10 +128,40 @@ const LogosSection = props => {
...
+          }
+        }
+      }
     }
   `)
-  const trustedByColumnCount = useMedia([2], 1)
 
+  const trustedByColumnCount = useMedia([2], 1)
+  const intl = useIntl()
+  const english_version = intl.locale === "en-US"
```
Comment (chochihim) [inline src/components/LogosSection/index.js]: use camel case to name variable

Comment (mailtime-wing) [reply src/components/LogosSection/index.js]: fixed.

File: src/components/LogosSection/index.js [hunk 2]
```diff
@@ -161,7 +161,7 @@ const LogosSection = props => {
 
   const trustedByColumnCount = useMedia([2], 1)
   const intl = useIntl()
-  const english_version = intl.locale === "en-US"
+  const English_version = intl.locale === "en-US"
```
Comment (chochihim) [inline src/components/LogosSection/index.js]: use camel case to name variable

Comment (mailtime-wing) [reply src/components/LogosSection/index.js]: changed.

**Pull request merged**